### PR TITLE
Gaurav_develop

### PR DESCRIPTION
**Pull request by gauravchauhan2403**
navbar is updated.

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -20,7 +20,7 @@ body {
 nav {
   width: 100%;
   text-align: right;
-  margin-top: 10px;
+  /* margin-top: 10px; */
```
Comment (swarajpure) [inline css/main.css:23]: It's a good practice not to keep commented code in our production code.

File: css/navbar.css [hunk 2]
```diff
@@ -4,6 +4,7 @@ nav ul {
   padding: 0;
   overflow: hidden;
   background-color: #1D1283;
+  /* border-bottom:.3rem solid #E30062 ; */
```
Comment (swarajpure) [inline css/navbar.css:7]: Same here. Commented code.

**Review by swarajpure — CHANGES_REQUESTED** [review]
Minor changes. Thank you! 😁

File: css/main.css (hunk 1 above)
Comment (gauravchauhan2403) [reply css/main.css:23]: Okay will keep this in mind from next  time.

Comment (gauravchauhan2403) [reply css/main.css]: Changes Have Been Done. Please Review And Merge.

**Review by swarajpure — APPROVED** [review]
Great. Thank you! 🎉

**Pull request merged**